lxc/copy.go

```diff
@@ -33,6 +33,46 @@ type cmdCopy struct {
 	flagAllowInconsistent bool
 }
 
+func expandProfile(destRemote lxd.InstanceServer, serverSideProfiles []string, deviceOverrides map[string]map[string]string) (map[string]map[string]string, error) {
```

Comment:
Can we use this function in `cmdInit.create` here https://github.com/canonical/lxd/blob/main/lxc/init.go#L303-L325?

Reply:
Yes, and I think here also https://github.com/canonical/lxd/blob/main/lxd/api_internal.go#L792-L815